frontend/src/modules/user/views/userProfileView.js

```diff
@@ -26,17 +23,18 @@ define(function(require){
 
     postRender: function() {
       this.setViewToReady();
+
+      this.$password = this.$('#password');
     },
 
     handleValidationError: function(model, error) {
       Origin.trigger('sidebar:resetButtons');
 
-      if (error && _.keys(error).length !== 0) {
-        _.each(error, function(value, key) {
-          this.$('#' + key + 'Error').text(value);
-        }, this);
-        this.$('.error-text').removeClass('display-none');
-      }
+      if (!error || _.keys(error).length === 0) return;
```

Comment:
```suggestion
      if (!error || !Object.keys(error).length) return;
```
Also suggest changing following `_.each()` to native `.forEach()`.